internal/common/common_unitest_db.go

```diff
@@ -119,13 +342,33 @@ func DeleteTestDB(db *gorm.DB, dbName string) {
 	Expect(db.Error).ShouldNot(HaveOccurred())
 }
 
-func openTestDB(dbName string) (*gorm.DB, error) {
-	dsn := fmt.Sprintf("host=127.0.0.1 port=%s user=postgres password=admin sslmode=disable", gDbCtx.GetPort())
+func getDBDSN(dbName string) string {
+	var host, port string
+	if os.Getenv("SKIP_UT_DB") != "" {
```

Comment:
Could be nice to put this logic inside getDBContext()

Reply:
Ah, good idea. Created `NoDBContext`